src/widgets/Posts/Posts.tsx

```
@@ -1,3 +1,45 @@
+  const debouncedValue = useDebounce(searchTerm, 500)
+  const { data: posts, refetch } = useGetAllPostsQuery({
+    variables: { endCursorPostId, searchTerm: debouncedValue },
+  })
+  const { data: newPostAdded } = usePostAddedSubscription()
+
+  const handleSearch = (value: string) => {
+    setSearchTerm(prevState => value)
+  }
+
+  useEffect(() => {
+    if (posts && posts?.getPosts.items.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)